packages/core/src/lib/icon/icon.svelte

```diff
@@ -64,6 +64,8 @@ const hasNameProperty = Object.hasOwn(paths, name);
 >
   <path
     d={paths[name]}
+    fill-rule="evenodd"
+    clip-rule="evenodd"
```

Comment:
Can you confirm that the clip-rule is necessary? After reading docs on MDN I think it is not.

Reply:
Thanks Jason, removed!